src/forms/FormFieldAdvancedSelection/LoadMoreButton.tsx

```diff
@@ -2,21 +2,28 @@ import * as React from "react";
 import { memo, ReactElement } from "react";
 import { LoadMoreButtonPropsTypes } from ".";
 import Button from "../../components/Button";
+import styled from "styled-components";
 
 const LoadMoreButton = (props: LoadMoreButtonPropsTypes): ReactElement => {
 	const { fieldDef } = props;
 
+	const ButtonWrapper = styled.div`

```

Comment:
Please also add 15px to the left and right sides of the button so it gets aligned with the other elements.